test/headless/specs/intersection-observer.js

```diff
@@ -79,6 +85,9 @@ testModule('IntersectionObserver', class extends TestClass {
       window.STATE.impressions = 0;
       let target = document.querySelector('.tracked-item[data-id="5"]');
       let observer = new spaniel.IntersectionObserver(function() {
+        var div = document.createElement('div');
+        div.id = 'impression-div';
+        document.body.appendChild(div);
```

Comment:
The actual test code shouldn't have to know about the DOM communication implementation details. Ideally, we would just call some function like `markImpression()` or something here.